ui/spinner.js

```diff
@@ -222,17 +222,14 @@ return $.widget( "ui.spinner", {
 		// button bindings
 		this.buttons = uiSpinner.find( "a" )
 			.attr( "tabIndex", -1 )
-			.button();
-
-			// Right now button does not support classes once it does adjust this with classes
-		this._removeClass( this.buttons, "ui-corner-all" );
+			.button({
+				"ui-button": ""
+			});
 
 		this._addClass( this.buttons.eq( 0 ), "ui-spinner-button ui-spinner-up" );
 		this._addClass( this.buttons.eq( 1 ), "ui-spinner-button ui-spinner-down" );
-		this._addClass( this.buttons.first().find( "span span" ), null,
-			"ui-icon " + this.options.icons.up );
-		this._addClass( this.buttons.last().find( "span span" ), null,
-			"ui-icon " + this.options.icons.down );
+		this.buttons.first().button( "option", "icon", this.options.icons.up );
```

Comment:
Should still switch to using `.eq()` here or to `.first()` above, same for `last`.